interface-definitions/firewall.xml.in

```diff
@@ -464,24 +464,39 @@
               </node>
             </children>
           </tagNode>
-          <leafNode name="interface">
+          <node name="member">
             <properties>
               <help>Interface associated with zone</help>
-              <valueHelp>
-                <format>txt</format>
-                <description>Interface associated with zone</description>
-              </valueHelp>
-              <valueHelp>
-                <format>vrf</format>
-                <description>VRF associated with zone</description>
-              </valueHelp>
-              <completionHelp>
-                <script>${vyos_completion_dir}/list_interfaces</script>
-                <path>vrf name</path>
-              </completionHelp>
-              <multi/>
             </properties>
-          </leafNode>
+            <children>
+              <leafNode name="interface">
```

Comment:
Please reference the generic interface include https://github.com/vyos/vyos-1x/blob/current/interface-definitions/include/generic-interface-multi.xml.i